app/buyers/views/buyers.py

```diff
@@ -270,59 +279,229 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+        ))
+
+        doc.add_style("intro", "table-column", (
+            TableColumnProperties(columnwidth="150pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("response", "table-column", (
+            TableColumnProperties(columnwidth="300pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("row", "table-row", (
+            TableRowProperties(useoptimalrowheight="true", breakbefore="auto"),
+        ))
+
+        sheet = doc.sheet("Supplier evidence")
+
+        questions = self.get_questions(brief['frameworkSlug'],
+                                       brief['lotSlug'],
+                                       'output_brief_response')
+
+        # two intro columns for boolean and dynamic lists
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
+
+        # HEADER
+        row = sheet.create_row("header")
+        row.write_cell(brief['title'], stylename="header",
+                       numbercolumnsspanned=str(len(responses) + 2))
+
+        # QUESTIONS
+        for question in questions:
+            if question.type in ('boolean_list', 'dynamic_list'):
+                length = len(brief[question.id])
+
+                row = sheet.create_row(question.id)
+                row.write_cell(question.name, stylename="title",
+                               numberrowsspanned=str(length + 1))
+
+                # TODO this is stupid, fix it (name needs to come from question)
+                row.write_cell("Requirement", stylename="subtitle")
+
+                for i, requirement in enumerate(brief[question.id]):
+                    row = sheet.create_row("{0}[{1}]".format(question.id, i))
+                    row.write_covered_cell()
+                    row.write_cell(requirement)
+            else:
+                row = sheet.create_row(question.id)
+                row.write_cell(question.name, stylename="title",
+                               numbercolumnsspanned="2")
+                row.write_covered_cell()
+
+        # RESPONSES
+        for response in responses:
+            sheet.create_column(stylename="response",
+                                defaultcellstylename="default")
+
+            for question in questions:
+                if question.type == 'dynamic_list':
+                    # TODO this is stupid too, fix it (name needs to come from question)
+                    row = sheet.get_row(question.id)
+                    row.write_cell("Evidence", stylename="subtitle")
+
+                    for i, item in enumerate(response[question.id]):
+                        row2 = sheet.get_row("{0}[{1}]".format(question.id, i))
+                        # TODO omg its everywhere, fix it (key needs to come from question)
+                        row2.write_cell(item.get('evidence') or '')
+
+                elif question.type == 'boolean_list':
```

Comment:
didn't take much to support them, it should support any kind of question type tbh as we dont know what will happen in the manifests going forward.